### PR TITLE
Adding `Refresh list` and `import tokens` link to the main page

### PR DESCRIPTION
**Pull request by NiranjanaBinoy**
This PR is a part of issue #11619.

This PR replaces the `Add Token` button from the main page with the `Import tokens` link and a related message.

Screen Shot 2021-08-19 at 8 55 10 PM

### CODE REVIEW

File: app/_locales/en/messages.json [hunk 1]
```diff
@@ -1290,6 +1290,9 @@
     "message": "verify the network details",
     "description": "Serves as link text for the 'mismatchedChain' key. This text will be embedded inside the translation for that key."
   },
+  "missingToken": {
+    "message": "Dont't see your token?"
```
Comment (ryanml) [inline app/_locales/en/messages.json:1294]: Should be `Don't see your token?`

**Review by ryanml — DISMISSED** [review]
lgtm, thanks for the good use of the design system! 🎉

**Review by darkwing — DISMISSED** [review]

**Review by brad-decker — DISMISSED** [review]

**Review by ryanml — DISMISSED** [review]
lgtm!

**Review by brad-decker — APPROVED** [review]

**Review by danjm — APPROVED** [review]
LGTM

**Pull request merged**